app/app.py

```
@@ -1,210 +1,70 @@
 from datetime import timedelta, datetime
-from flask import Flask, render_template, request, redirect, url_for, session, flash
+from flask import Flask
+from flask_login import LoginManager
 from flask_mysqldb import MySQL
```

Ruff (0.8.2)

`flask_mysqldb.MySQL` imported but unused

Remove unused import: `flask_mysqldb.MySQL`

(F401)